prov/verbs/src/ep_rdm/verbs_ep_rdm.c

```diff
@@ -554,25 +562,49 @@ int fi_ibv_rdm_open_ep(struct fid_domain *domain, struct fi_info *info,
 		}
 	}
 
+	_ep->rq_wr_depth = info->rx_attr->size;
+	_ep->sq_wr_depth = _ep->n_buffs + 1;
+	if (!fi_param_get_str(&fi_ibv_prov, "rdm_eager_send_opcode", &str_param))
+	{
+		if (!strncmp(str_param, "IBV_WR_RDMA_WRITE_WITH_IMM",
+			     strlen("IBV_WR_RDMA_WRITE_WITH_IMM")))
+		{
+			_ep->eopcode = IBV_WR_RDMA_WRITE_WITH_IMM;
+		} else if (!strncmp(str_param, "IBV_WR_SEND",
+				    strlen("IBV_WR_SEND")))
+		{
+			_ep->eopcode = IBV_WR_SEND;
+		} else {
+			FI_INFO(&fi_ibv_prov, FI_LOG_CORE,
+				"invalid value of rdm_eager_send_opcode\n");
+			ret = -FI_EINVAL;
+			goto err;
+		}
+	} else {
+		_ep->eopcode = IBV_WR_SEND;
+	}
```

Comment:
@shefty I would keep this WRITE_WITH_IMM way for several reasons:

1. it was better tested, see the [comment](https://github.com/ofiwg/libfabric/pull/2403/files#diff-ebe09d747e2d50112ca4ba8bf2e36b9eR585), it may cause broken ordering in some cases.
2. I do not know all side effects from performance perspective for more complex tests like collectives. For example, message rate is a bit worse with SEND.

I think if we will have some perf overhead on memory touching, we may make these ways independent on initialization.